src/k8s/pkg/k8sd/app/hooks_bootstrap.go

```diff
@@ -178,11 +178,19 @@ func (a *App) onBootstrapWorkerNode(ctx context.Context, s state.State, encodedT
 		return fmt.Errorf("failed to write worker node certificates: %w", err)
 	}
 
+	// We don't have the CIDRs on the workers, hence we check the address type of the control-plane node.
+	var localhostAddress string
+	if utils.IsIPv4(address) {
+		localhostAddress = "127.0.0.1"
```

Comment:
we might add consts for these:
```go
var (
    IPv4Loopback string = "127.0.0.1"
    IPv6Loopback string = fmt.Sprintf("[%s]", net.IPv6loopback)
)
```

Reply:
I'd say we can be explicit. Those addresses are not expected to change ever and string assembly makes it slightly harder to read.
IFF we would ever need to change that addresses then IMHO a search-replace is as fast as changing the const.